.changes/next-release/bugfix-AmazonDynamoDB-965386f.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "Amazon DynamoDB",
```

Comment:
Should change to "Amazon DynamoDB Enhanced Client"

Reply:
Updated and pushed